lib/packages/core/src/api/factories/account/getAccountTransactions.ts

```diff
@@ -37,5 +38,5 @@ export const getAccountTransactions = (service: BurstService):
             numberOfConfirmations,
             type,
             subtype,
-            includeIndirect
+            includeIndirect: service.settings.brsVersion === BrsVersion.BEFORE_V2_3 ? undefined : includeIndirect
```

Comment:
Seeing as how `includeIndirect` is already optional, it might be cleaner if the app handles this logic.

Reply:
Yeah. You're right, but then a dev still needs to know that indirect must be undefined for older versions. So, the version wouldn't be coupled with the interface at all. Is cleaner, but less dev-friendly imho

Reply:
Oh, wait.... sleeping one night, and with fresh mind, I see that I did bullshit. Will close this PR